vowpalwabbit/core/src/scored_config.cc

```diff
@@ -38,6 +38,10 @@ void scored_config::reset_stats(double alpha, double tau)
   update_count = 0;
 }
 
+float scored_config::lower_bound() { return chisq.lower_bound(); }
+
+float scored_config::upper_bound() { return current_ips(); }
```

Comment:
I'm guessing this is temporary

Reply:
Yes this should be temporary until we find something better for upper_bound